imap_processing/cdf/config/imap_idex_l1b_variable_attrs.yaml

```
@@ -0,0 +1,330 @@
+int_fillval: &int_fillval -9223372036854775808
+
+max_uint16: &max_uint12 4095
```

I think that these names are intended to match?
```suggestion
max_uint12: &max_uint12 4095
```

---

Good catch